ddtrace/internal/processor/trace.py

```diff
@@ -226,3 +227,34 @@ def shutdown(self, timeout):
         except ServiceStatusError:
             # It's possible the writer never got started in the first place :(
             pass
+
+
+@attr.s
+class SingleSpanSamplingProcessor(SpanProcessor):
+    """SpanProcessor for sampling single spans:
+    - Users can specify span rules with service, name, sample rate, and rate limit.
+    - Those rules are used to assess all spans that are not sampled by trace sampling.
+    If the rules assess that a span should be kept, we add specific tags to it.
+    - The single spans that will be kept are sent in separate payloads if stats computation is enabled.
+    If stats computation is not enabled, then the single spans will be sent along with normal trace payloads.
+    """
```

Comment:
```suggestion
    """SpanProcessor for sampling single spans:
    
    - Users can specify span rules with service, name, sample rate, and rate limit.
    - Those rules are used to assess all spans that are not sampled by trace sampling.
      If the rules assess that a span should be kept, we add specific tags to it.
    - The single spans that will be kept are sent in separate payloads if stats computation is enabled.
    
    If stats computation is not enabled, then the single spans will be sent along with normal trace payloads.
    """
```

reformatting so it'll render properly as rST